src/app/models/evt-models.ts

```diff
@@ -170,6 +170,68 @@ export class ApparatusEntry extends GenericElement {
     nestedAppsIDs: string[];
 }
 
+export const SourceClass = 'sourceEntry';
+export const AnalogueClass = 'analogueEntry';
+export const BibliographyClass = 'biblioEntry';
+
+export class QuoteEntry extends GenericElement {
+    id: string;
+    tagName: string;
+    text: string;
+    sources: BibliographicEntry[]|BibliographicList;
+    extSources: BibliographicEntry[];
+    extElements: any;
+    analogues: any;
+    originalEncoding: string;
+    insideCit: boolean;
+    quotedText: string[];
+    noteView: boolean;
+    contentToShow: Array<ParseResult<GenericElement>>
+    //rend: string;
+}
+
+export class BibliographicList extends GenericElement {
+    id: string;
+    head: string[];
+    sources: BibliographicEntry[];
+    //TODO: biblStruct
+}
+
+export class BibliographicEntry extends GenericElement {
+    id: string;
+    author: string[];
+    editor: string[];
+    title: string[];
+    date: string[];
+    publisher: string[];
+    pubPlace: string[];
+    citedRange: string[];
+    biblScope: string[];
+    text: string;
+    quotedText: string;
+    insideCit: boolean;
+    originalEncoding: string;
```

Comment:
Ok i took as a model ApparatusEntry because I was first requested to show that element in the frame and there originalEncoding type was a string. Should i change it there too? (evt-models.ts line 169)